pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -208,13 +208,16 @@ stdenv.mkDerivation {
     + ''
       export named_cc=${targetPrefix}cc
       export named_cxx=${targetPrefix}c++
+      export needsTarget=0
 
       if [ -e $ccPath/${targetPrefix}gcc ]; then
         wrap ${targetPrefix}gcc $wrapper $ccPath/${targetPrefix}gcc
         ln -s ${targetPrefix}gcc $out/bin/${targetPrefix}cc
         export named_cc=${targetPrefix}gcc
         export named_cxx=${targetPrefix}g++
       elif [ -e $ccPath/clang ]; then
+        needsTarget=1
+        export clangTarget=${targetPlatform.config}
```

Comment:
Cause the other rename I would call this `defaultTarget` too.

Reply:
will fix

Reply:
updated